### ISSUE TITLE
Possible to change form background color?

### ISSUE THREAD

**Issue by brxg** [tlovett1/custom-contact-forms#82]
Hi, 

To start, I am not a programmer.  Can you please provide an ELI5 answer to my question below?

I really like the versatility of this WP PlugIn.  However, my site's overall background is white, like Google's.  So, unfortunately, the contact form's input fields are washed out on my Contact page on my site.  Is there any way to change the background color of just the contact form itself such that the input fields will remain white but I can make the background color of the contact form itself something like a light grey so there will be enough contrast that my site visitors can easily see the input fields within the contact form?

Thanks for your time and help!  

I use WP and a theme and have very basic experience changing some of the html, but my theme only let's me adjust the background for my entire site.  I haven't figured out how to change only the background within the contact form, and I've reviewed all your materials and don't see any reference to this idea.  If it's a function that can be provided within the plugin itself it may help get around my theme's limitations, or at least that's my hope.

Thanks

**Comment by tlovett1** [comment]
Unfortunately, you will have to style the form with your theme.

**Comment by brxg** [comment]
I understand, and thought I would need to give up.  But, out of curiosity I
did as you said and adjusted my theme's background color to a very light
grey (f4f4f4) and I'm amazed at how much better the entire site looks.  I
didn't realize how glaring the all white background really is.  So, if
anyone else asks about this, I'd suggest they take the plunge and try going
from a white background to something very light.  It will help reduce glare
And your contact form fields now have enough contrast to be noticeable.

Thanks for the super quick reply too.
